app/components/ItemList/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+*
+* ItemList
+*
+*/
+
+import React from 'react';
+// import styled from 'styled-components';
+import { Table } from 'react-bootstrap';
+import PropType from 'prop-types';
+import NoImgFound from 'images/no_image_available.png';
+
+class ItemList extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  constructor(props) {
+    super(props); // headervals len should = todisplay
+    this.genList = this.genList.bind(this);
+    this.genItem = this.genItem.bind(this);
+    this.picKeys = [];
+    this.toDisplay = this.genList(this.props.data, this.picKeys);
+  }
+  genList(dataArray, picKeys) {
+    const toRender = [];
+    for (let i = 0, len = dataArray.length; i < len; i += 1) {
+      toRender.push(this.genItem(dataArray[i], picKeys));
+    }
+    return <tbody>{toRender}</tbody>;
+  }
+  genItem(obj, picKeys) {
+    const keys = Object.keys(obj);
+    let pic = null;
+    const items = [];
+    for (let i = 0, len = keys.length; i < len; i += 1) {
+      const key = keys[i];
+      if (obj[key] && obj[key].toString().slice(0, 4) === 'http') {
+        pic = obj[key];
+        picKeys.push(key);
+      } else if (!picKeys.includes(key)) { // don't display keys proven to be pics
+        items.push(
+          <Table style={{ padding: 0, margin: 0 }}>
+            <tbody>
+              <tr>
+                <td style={{ width: '30%' }}><strong>{key}:</strong></td>
+                <td>{obj[key] || ''}</td>
```

Comment:
There is a bug around here preventing the data to be displayed. I was able to get it to display the data by moving  the stuff within braces from line 43 to right after the closing strong tag on line 42. Need further validation that this fixes it